src/execute_request.jl

```diff
@@ -79,7 +79,11 @@ function execute_request(socket, msg)
 
         user_expressions = Dict()
         for (v,ex) in msg.content["user_expressions"]
-            user_expressions[v] = invokelatest(parse, ex)
+            value = SOFTSCOPE[] ? softscope_include_string(current_module[], code) :
+                                  include_string(current_module[], code)
```

Comment:
I would just use `include_string` here.    Since the `user_expressions` here are a low-level thing used for communication between the front-end and the kernel, they shouldn't be affected by the users's `SOFTSCOPE` setting.

Reply:
Shouldn't this be evaluating `ex`, not `code`?

Reply:
Whoops, it absolutely should be evaluating `ex`.  And sounds good about blanked disabling softscope here.